package.json

```diff
@@ -1118,56 +1118,94 @@
             "media": {
               "image": "resources/walkthrough/connect.png",
               "altText": "Connect to Kafka clusters"
             },
             "completionEvents": [
-              "onCommand:command:confluent.connections.ccloud.signIn",
-              "onCommand:confluent.docker.startLocalResources"
+              "onCommand:command:confluent.connections.ccloud.signIn"
             ]
           },
           {
             "id": "create-connections",
-            "title": "Connect directly to Kafka / Schema Registry",
-            "description": "To connect to an Apache Kafka or Schema Registry cluster, click $(plus) next to Resource > Other. The cluster can be in Confluent Cloud, Confluent Platform, or hosted elsewhere.\n[Open Connection Form](command:confluent.connections.direct)",
+            "title": "Connect Directly",
+            "description": "To connect to Confluent Cloud (via API Credential), Confluent Platform, WarpStream, Apache Kafka, and/or Schema Registry clusters.\n Select the $(plus) icon next to Resource to create or import a connection.\n[Create Connection](command:confluent.connections.direct)",
             "media": {
               "image": "resources/walkthrough/direct-connection-form.png",
-              "altText": "Create a new connection to a Kafka cluster or Schema Registry"
+              "altText": "Create a new connection to a Kafka cluster and/or Schema Registry"
             },
             "completionEvents": [
               "onCommand:confluent.connections.direct"
             ]
           },
+          {
+            "id": "connect-local-cluster",
+            "title": "Start Local Environment",
+            "description": "If you have Kafka running locally, it appears automatically in the Resources view.\nYou can also start Kafka and Stream Registry clusters using the extension.\n[Start Local Resources](command:confluent.docker.startLocalResources)",
+            "media": {
+              "image": "resources/walkthrough/start-local.png",
+              "altText": "Connect to Kafka clusters"
+            },
+            "completionEvents": [
+              "onCommand:confluent.docker.startLocalResources"
+            ]
+          },
           {
             "id": "view-topics",
             "title": "View Messages",
-            "description": "Click the $(confluent-view-messages) icon next to a topic to open the Message Viewer and explore the messages produced to the topic",
+            "description": "Select the $(confluent-view-messages) icon next to a topic to open the Message Viewer and explore the messages produced to the topic.",
             "media": {
               "image": "resources/walkthrough/message-viewer.png",
-              "altText": "Click the View button next to a topic to see the Message viewer."
+              "altText": "Select the View button next to a topic to see the Message viewer."
```

Comment:
```suggestion
              "altText": "Select the 'View Messages' button next to a topic to see the Message viewer."
```

Reply:
Updated but referenced the icon instead of text.